src/Linux/R/Interpreters/Test/RInstallationTest.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.Common.Core;
+using Microsoft.UnitTests.Core.Linux;
+using Xunit;
+
+namespace Microsoft.R.Interpreters.Linux.Test {
+    public class RInstallationTest {
+        [Fact]
```

Comment:
> `[Fact]` was the default for .Net Core test project.

Yes it is, though we use our own attributes. They are defined in .net framework project, so you won't be able to use them. I've created a task #3489